### ISSUE TITLE
Embedded MultipleInput with enableError on new rows

### ISSUE THREAD

**Issue by Lordfef** [unclead/yii2-multiple-input#100]
Hi, compliments for this widget.
I'm trying embedded multipleinput like this

```
        <?= \unclead\widgets\MultipleInput::widget([
            'model' => $wizmodel,
            'attribute' => 'wgs',
            'attributeOptions' => [
                'enableAjaxValidation'      => true,
                'enableClientValidation'    => true,
                'validateOnChange'          => true,
                'validateOnSubmit'          => true,
                'validateOnBlur'            => true,
            ],
            'columns' => [
                [
                    'name' => 'id_group',
                    'type' => \unclead\widgets\MultipleInputColumn::TYPE_HIDDEN_INPUT
                ],

                [
                    'name' => 'name',
                    'title' => 'name',
                    'enableError' => true,
                ],
                [
                    'name' => 'subject',
                    'type' => MultipleInput::class,
                    'options' => [
                        'attributeOptions' => [
                            'enableAjaxValidation'      => true,
                            'enableClientValidation'    => true,
                            'validateOnChange'          => true,
                            'validateOnSubmit'          => true,
                            'validateOnBlur'            => true,
                        ],
                        'columns' => [
                            [
                                'name'  => 'id_subject',
                                'type'  => MultipleInputColumn::TYPE_DROPDOWN,
                                'title' => 'subject',
                                //'defaultValue' => 33,
                                'items' =>  [
                                    31 => 'item 31',
                                    32 => 'item 32',
                                    33 => 'item 33',
                                    34 => 'item 34',
                                    35 => 'item 35',
                                    36 => 'item 36',
                                ],
                                'options' => [
                                    'prompt'=>'select subject',
                                ],
                            ]
                        ]
                    ]
                ]
            ],
        ]); ?>
```

and this is my post parameter:
```
WizGroupSubject[wgs][0][id_group]:""
WizGroupSubject[wgs][0][name]:"xxx"
WizGroupSubject[wgs][0][subject][id_subject][0]:"31"
WizGroupSubject[wgs][0][subject][id_subject][1]:"32"
WizGroupSubject[wgs][0][subject][id_subject][2]:"33"
WizGroupSubject[wgs][1][id_group]:""
WizGroupSubject[wgs][1][name]:"yyy"
WizGroupSubject[wgs][1][subject][id_subject][0]:"34"

```
now i want to enableError for the field 'id_subject' and i see that i must specify a model for the embedded class.

So i try this:

```
<?= \unclead\widgets\MultipleInput::widget([
            'model' => $wizmodel,
            'attribute' => 'wgs',
            'attributeOptions' => [
                'enableAjaxValidation'      => true,
                'enableClientValidation'    => true,
                'validateOnChange'          => true,
                'validateOnSubmit'          => true,
                'validateOnBlur'            => true,
            ],
            'columns' => [
                [
                    'name' => 'id_group',
                    'type' => \unclead\widgets\MultipleInputColumn::TYPE_HIDDEN_INPUT
                ],

                [
                    'name' => 'name',
                    'title' => 'name',
                    'enableError' => true,
                ],
                [
                    'name' => 'subject',
                    'type' => MultipleInput::class,
                    'options' => [
                        'attributeOptions' => [
                            'enableAjaxValidation'      => true,
                            'enableClientValidation'    => true,
                            'validateOnChange'          => true,
                            'validateOnSubmit'          => true,
                            'validateOnBlur'            => true,
                        ],
                        'model' => $wizmodel,
                        'attribute' => 'wgs',
                        'columns' => [
                            [
                                'name'  => 'id_subject',
                                'type'  => MultipleInputColumn::TYPE_DROPDOWN,
                                'title' => 'subject',
                                //'defaultValue' => 33,
                                'enableError' => true,
                                'items' =>  [
                                    31 => 'item 31',
                                    32 => 'item 32',
                                    33 => 'item 33',
                                    34 => 'item 34',
                                    35 => 'item 35',
                                    36 => 'item 36',
                                ],
                                'options' => [
                                    'prompt'=>'select subject',
                                ],
                            ]
                        ]
                    ]
                ]
            ],
        ]); ?>

````
with this result:
```
WizGroupSubject[wgs][0][id_group]:""
WizGroupSubject[wgs][0][name]:"xxx"
WizGroupSubject[wgs][0][subject][id_subject][0]:"31"
WizGroupSubject[wgs][0][subject][id_subject][1]:"32"
WizGroupSubject[wgs][0][subject][id_subject][2]:"33"
WizGroupSubject[wgs][1][id_group]:""
WizGroupSubject[wgs][1][name]:"yyy"
WizGroupSubject[wgs][id_subject][0]:"34"
```

As you can see this last row has a problem. It's my error or ther's a bug?
The model is a simple model, not an AR.
thank's

**Comment by unclead** [comment]
You don't need to pass a model to the embedded widget because it passed automatically here: https://github.com/unclead/yii2-multiple-input/blob/master/src/MultipleInputColumn.php#L148

Try to use a workaround from #101

**Comment by unclead** [comment]
fixed via 3b597aff45a9dabd92b99846d023ca3c37c10a17
will be release in 2.0 (soon)
